packages/aws-rfdk/lib/core/scripts/bash/mountEfs.sh

```diff
@@ -13,12 +13,15 @@
 #  $1 -- EFS Identifier (ex: efs-00000000000)
 #  $2 -- Mount path; directory that we mount the EFS to.
 #  $3 -- (optional) NFSv4 mount options for the EFS. 
+#  $4 -- (optional) whether to obtain the EFS mount target's IP address using the EFS API and persist this to
+#        the /etc/hosts file on the system. This allows the script to work when the mounting instance cannot resolve the
+#        mount target using DNS. This defaults to being disabled, specify "true" to enable this feature.
 
 set -xeu
 
 if test $# -lt 2
 then
-  echo "Usage: $0 <file system ID> <mount path> [<mount options>]"
+  echo "Usage: $0 FILE_SYSTEM_ID MOUNT_PATH [MOUNT_OPTIONS] [RESOLVE_MOUNT_POINT_USING_API]"
```

Comment:
Makes sense to me to make the new arg required. Two optional positional args can lead to confusion.